docs/_source/components-overview/pop_syn/single_star.rst

```diff
@@ -21,85 +23,167 @@ STARPROPERTIES
+  * - ``log_R``
+    - log10 stellar radius in R_sun.
+  * - ``log_L``
+    - log10 surface stellar luminosity in L_sun.
+  * - ``mdot``
+    - Stellar mass-loss rate in Msun/yr.
+  * - ``lg_wind_mdot``
+    - log10 stellar wind mass-loss rate in Msun/yr.
+  * - ``he_core_mass``
+    - Helium core mass in M_sun.
+  * - ``he_core_radius``
+    - Helium core radius in R_sun.
+  * - ``c_core_mass``
+    - Carbon core mass in M_sun.
+  * - ``c_core_radius``
+    - Carbon core radius in R_sun.
+  * - ``o_core_mass``
+    - Oxygen core mass in M_sun.
+  * - ``o_core_radius``
+    - Oxygen core radius in R_sun.
+  * - ``center_h1``
+    - Hydrogen central mass fraction abundance.
+  * - ``center_he4``
+    - Helium central mass fraction abundance.
+  * - ``center_c12``
+    - Carbon central mass fraction abundance.
+  * - ``center_n14``
+    - Nitrogen central mass fraction abundance.
+  * - ``center_o16``
+    - Oxygen central mass fraction abundance.
+  * - ``surface_h1``
+    - Hydrogen surface mass fraction abundance.
+  * - ``surface_he4``
+    - Helium surface mass fraction abundance.
+  * - ``surface_c12``
+    - Carbon surface mass fraction abundance.
+  * - ``surface_n14``
+    - Nitrogen surface mass fraction abundance.
+  * - ``surface_o16``
+    - Oxygen surface mass fraction abundance.
+  * - ``log_LH``
+    - log10 total thermal power from PP and CNO, excluding neutrinos devided by L_sun.
+  * - ``log_LHe``
+    - log10 total thermal power from triple-alpha, excluding neutrinos devided by L_sun.
+  * - ``log_LZ``
+    - log10 total burning power excluding LH and LHe and photodisintegrations devided by L_sun.
+  * - ``log_Lnuc``
+    - log10 total nuclear reaction luminosity (LH + LHe LZ) in L_sun.
+  * - ``c12_c12``
+    - log10 total luminosity for c12_c12 reaction in L_sun.
+  * - ``surf_avg_omega_div_omega_crit``
+    - Average surface omega divided by critical omega.
+  * - ``total_moment_of_inertia``
+    - Total moment of inertia in g*cm^2.
+  * - ``log_total_angular_momentum``
+    - log10 total angular momentum of the star g*cm^2*s^-1
+  * - ``spin``
+    - Angular momentum of the star in g*cm^2*s^-1 or dimensionless BH spin.
+  * - ``profile``
+    - Stellar profile from MESA. [not current supported for the initial-final interpolator]
 
 Additional scalar properties are added during the evolution depending on which steps the star has undergone. These properties are not stored in the history.
 
-.. csv-table:: Additional output
-   :header: "Properties", "Descriptions"
-   :widths: 50, 150
-  * - `natal_kick_array`
-    - The natal kick array for the star if it has undergone a SN.
-      [velocity, theta, phi, ?]
-  * - `SN_type`
+.. list-table:: Additional output
+   :header-rows: 1
+   :widths: 50 150
+
+  * - Properties
+    - Descriptions
+  * - ``natal_kick_array``
+    - | The natal kick array for the star if it has undergone a SN. 
+      | contains:
+      
+      * velocity
+      * theta
+      * phi
+      * ?
```

Comment:
The fourth quantity should be the mean anomaly.

Reply:
Added :)